### PR TITLE
improve onboarding flow

### PR DESCRIPTION
**Pull request by Himanshu-Singh-Chauhan**
#trypear/pearai-app#157

changes:
- add pear to welcome page
- onboarding flow change = Cloud Models -> Add new model page
- refactor `onboarding.tsx` for better readability.
- fix button sizes and text consistency.
- add pear logo on login button.

https://github.com/trypear/pearai-submodule/assets/26621232/3cfb3896-ca8d-4763-a0e3-078b6e1725be

### CODE REVIEW

**Comment by nang-dev** [comment]
this is amazing

**Review by nang-dev — APPROVED** [review]

**Comment by nang-dev** [comment]
Screenshot 2024-07-07 at 5 47 46 PM


Yo @Himanshu-Singh-Chauhan  i think it theres still a style issue when the button is too small.

taking a look though

File: gui/src/pages/models.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ function Models() {
         <ArrowLeftIcon
           width="1.2em"
           height="1.2em"
-          onClick={() => navigate("/")}
+          onClick={() => navigate("/onboarding")}
```
Comment (MaySoMusician) [inline gui/src/pages/models.tsx:52]: This should be left `"/"` if we open this page after a user has completed onbording (i.e. by clicking "+" button on the bottom on the main chat page)

Comment (nang-dev) [reply gui/src/pages/models.tsx:52]: correct whoops

**Comment by nang-dev** [comment]
> Screenshot 2024-07-07 at 5 47 46 PM
> 
> Yo @Himanshu-Singh-Chauhan i think it theres still a style issue when the button is too small.
> 
> taking a look though

I fixed it here #86

File: gui/src/pages/models.tsx (hunk 1 above)
Comment (nang-dev) [reply gui/src/pages/models.tsx:52]: done #86

**Pull request merged**